app/pages/5_Model_assumptions.py

```diff
@@ -0,0 +1,83 @@
+
+import streamlit as st
+import pandas as pd
+import geopandas as gpd
+import plotly.subplots as sp
+import plotly.graph_objects as go
+
+st.title("Modelling assumptions")
+
+st.header("Assumptions for technology costs")
+
+st.markdown(
+    "Model assumptions are under constant revision."
+    "Assumptions used for this study are basted on the "
+    "[technology-data repository v0.6.2](https://github.com/PyPSA/technology-data/blob/master/outputs/costs_2030.csv). "
```

Comment:
Would be probably nice to explain a bit what is actually that repo. I think, something like this would do: `PyPSA database on technology costs which accounts for costs evolution at 2030`. What is your feeling about that?

Typos may be fixed, as well (a whitespace between the phrases and `based`) ;)